src/features/vote/helpers.ts

```diff
@@ -1,35 +1,46 @@
 import { DateTime } from "luxon";
-import { PriceRound } from "web3/queryVotingContractEvents";
 import { ethers } from "ethers";
+import web3 from "web3";
 import { PriceRequestRound } from "common/hooks/useVoteData";
 import { queryRetrieveRewards } from "web3/queryVotingContractMethods";
 import { PastRequest } from "./PastRequests";
+import { PostCommitVote } from "web3/postVotingContractMethods";
+import { PendingRequest } from "web3/queryVotingContractMethods";
+import stringToBytes32 from "common/utils/web3/stringToBytes32";
+import EthCrypto from "eth-crypto";
+import {
+  computeVoteHashAncillary,
+  getRandomSignedInt,
+  encryptMessage,
+} from "common/tempUmaFunctions";
 
-const ACTIVE_DAYS_CONSTANT = 2.5;
-
-export function isActiveRequest(round: PriceRound) {
-  const currentTime = DateTime.local();
-  const roundTime = DateTime.fromSeconds(Number(round.time));
-  const diff = currentTime.diff(roundTime, ["days"]).toObject();
-  const { days } = diff;
-  if (days) {
-    return days > 0 && days <= ACTIVE_DAYS_CONSTANT ? true : false;
-  } else {
-    return false;
-  }
-}
+import { FormData } from "./ActiveRequestsForm";
 
-export function isPastRequest(round: PriceRound) {
-  const currentTime = DateTime.local();
-  const roundTime = DateTime.fromSeconds(Number(round.time));
-  const diff = currentTime.diff(roundTime, ["days"]).toObject();
-  const { days } = diff;
-  if (days) {
-    return days > 0 && days > ACTIVE_DAYS_CONSTANT ? true : false;
-  } else {
-    return false;
-  }
-}
+// const ACTIVE_DAYS_CONSTANT = 2.5;
+
+// export function isActiveRequest(round: PriceRound) {
+//   const currentTime = DateTime.local();
+//   const roundTime = DateTime.fromSeconds(Number(round.time));
+//   const diff = currentTime.diff(roundTime, ["days"]).toObject();
+//   const { days } = diff;
+//   if (days) {
+//     return days > 0 && days <= ACTIVE_DAYS_CONSTANT ? true : false;
+//   } else {
+//     return false;
+//   }
+// }
+
+// export function isPastRequest(round: PriceRound) {
+//   const currentTime = DateTime.local();
+//   const roundTime = DateTime.fromSeconds(Number(round.time));
+//   const diff = currentTime.diff(roundTime, ["days"]).toObject();
+//   const { days } = diff;
+//   if (days) {
+//     return days > 0 && days > ACTIVE_DAYS_CONSTANT ? true : false;
+//   } else {
+//     return false;
+//   }
+// }
```

Comment:
why did this get commented out

Reply:
Oh, I should just delete it. This is not the way to get active and past rounds.